openassessment/xblock/test/test_submission.py

```diff
@@ -774,6 +867,35 @@ def test_open_saved_response_old_format(self, xblock):
             }
         )
 
+    @scenario('data/team_submission.xml', user_id="Red Five")
+    def test_team_open_submitted(self, xblock):
+        """ When a submission is created for a team, we create identical submissions for each learner.
+            Since we can't save submission info to other learners state, we need to query the database
+            on page load to see if a submisison has been created by a team member.
+        """
+        SubmissionTest.setup_mock_team(xblock)
+        submissions = xblock.create_team_submission(
+            ('A man must have a code', 'A man must have an umbrella too.')
+        )
+        # get submission from Xblock
+        submission = [_ for _ in submissions if _['uuid'] == xblock.submission_uuid][0]
```

Comment:
You should only use `_` as a variable name when you really don't need it.  In here, you need it for the conditional portion of your list comprehension.  Maybe just `sub` instead?